CHANGELOG.md

```diff
@@ -46,7 +46,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Bump `com.networknt:json-schema-validator` from 1.0.78 to 1.0.81 (#7460)
 - Bump Apache Lucene to 9.6.0 ([#7505](https://github.com/opensearch-project/OpenSearch/pull/7505))
 - Bump `com.google.cloud:google-cloud-core-http` from 1.93.3 to 2.17.0 (#7488)
-- Bump `com.google.guava:guava` from 30.1.1-jre to 31.1-jre (#7565)
+- Bump `com.google.guava:guava` from 30.1.1-jre to 32.0.0-jre (#7565, #7811)
```

Comment:
Ugh. This will conflict with #7841.
I also merged in https://github.com/opensearch-project/OpenSearch/pull/7816 which will cause conflicts for #7841. 
Let's hold off on the merge until we clear out the changelog?

Reply:
Sure @kotwanikunal , please feel free to hold on with approval (I have like 5 other backports to do but I won't open pull requests until we get this one in), thanks!